src/github3/actions/secrets.py

```diff
@@ -0,0 +1,227 @@
+
+    See https://docs.github.com/en/rest/actions/secrets for more details.
+    GitHub never reveals the secret value through its API, it is only accessible
+    from within actions. Therefore, this object represents the secret's metadata
+    but not its actual value.
+    """
+
+    class_name = "_Secret"
+
+    def _repr(self):
+        return f"<{self.class_name} [{self.name}]>"
+
+    def __str__(self):
+        return self.name
+
+    def _update_attributes(self, secret):
+        self.name = secret["name"]
+        self.created_at = self._strptime(secret["created_at"])
+        self.updated_at = self._strptime(secret["updated_at"])
+
+
+class RepositorySecret(_Secret):
+    """An object representing a repository secret for GitHub Actions.
+
+    See https://docs.github.com/en/rest/actions/secrets for more details.
+    GitHub never reveals the secret value through its API, it is only accessible
+    from within actions. Therefore, this object represents the secret's metadata
+    but not its actual value.
+
+    .. attribute:: name
+
+        The name of the secret
+
+    .. attribute:: created_at
+
+        The timestamp of when the secret was created
+
+    .. attribute:: updated_at
+
+        The timestamp of when the secret was last updated
+    """
+
+    class_name = "RepositorySecret"
+
+
+class SharedOrganizationSecret(_Secret):
+    """An object representing an organization secret for GitHub Actions that is
+    shared with the repository.
+
+    See https://docs.github.com/en/rest/actions/secrets for more details.
+    GitHub never reveals the secret value through its API, it is only accessible
+    from within actions. Therefore, this object represents the secret's metadata
+    but not its actual value.
+
+    .. attribute:: name
+
+        The name of the secret
+
+    .. attribute:: created_at
+
+        The timestamp of when the secret was created
+
+    .. attribute:: updated_at
+
+        The timestamp of when the secret was last updated
+    """
+
+    class_name = "SharedOrganizationSecret"
+
+
+class OrganizationSecret(_Secret):
+    """An object representing am organization secret for GitHub Actions.
+
+    See https://docs.github.com/en/rest/actions/secrets for more details.
+    GitHub never reveals the secret value through its API, it is only accessible
+    from within actions. Therefore, this object represents the secret's metadata
+    but not its actual value.
+
+    .. attribute:: name
+
+        The name of the secret
+
+    .. attribute:: created_at
+
+        The timestamp of when the secret was created
+
+    .. attribute:: updated_at
+
+        The timestamp of when the secret was last updated
+    """
+
+    class_name = "OrganizationSecret"
+
+    def _update_attributes(self, secret):
+        super()._update_attributes(secret)
+        self.visibility = secret["visibility"]
+        if self.visibility == "selected":
+            self._selected_repos_url = secret["selected_repositories_url"]
```

Comment:
Why is the URL attribute "private"? It should be public and named identically to the value in the JSON.

Reply:
True, I made it public and renamed it to `selected_repositories_url`.